InventoryComponents/InventoryHUD/InventoryHudRelicComponent.gd

```diff
@@ -3,20 +3,33 @@ class_name InventoryHUDRelicComponent
 
 var relic_display : PackedScene = preload("res://InventoryComponents/InventoryHUD/relic_display.tscn")
 var relic_display_array : Array[InventoryHUDRelicDisplay]
-@export var relic_start_pos : Node
+@onready var relic_start_pos : Node = $StartPos
 @export var relic_space : float
+var _relic_array : Array[Relic] = []
 
-func update_relic_displays(relics_array : Array[ Relic]) -> void:
+func _ready() -> void:
+	_relic_array += InventoryManager.relic_component.held_relics
+	update_relic_display(null, true)
+
+
+func update_relic_display(relic : Relic, is_added : bool) -> void:
```

Comment:
what is the goal of having a function that takes a relic to add it to the display if you only give it `null` as a relic ?

Reply:
It's to update the relic display, I'll drag that part to another function and just call that